src/applications/form-renderer/_config/formConfig.js

```diff
@@ -138,6 +138,15 @@ export const normalizedForm = {
         includeDateOfBirth: false,
       },
     },
+    {
+      id: 160592,
+      chapterTitle: 'Generated Identification Information',
+      type: 'digital_form_identification_info',
+      pageTitle: 'Identification Information',
+      additionalFields: {
+        includeServiceNumber: false,
+      },
+    },
```

Comment:
Can you refresh me on your workflow, in general? How are you utilizing this `normalizedForm` data? What purpose is it serving?

I have the same question for the `employmentQuestionnaire` object below, though I can understand that it serves as a blueprint for what we want `content-build` to output. So are you creating this object here in `vets-website`, making sure it renders appropriately, then, if so, ensuring that `content-build` is outputting things to match? That makes sense, but it raises the question for me how you're toggling this application to read the static value rather than from the KISS file. I know the `fetchFormConfig` function _can_ take a parameter, but I don't see the calling context having code that is variable in any way. Are you simply temporarily hard-coding the change for testing/verification?

Note that these are merely questions for my own understanding. I am completely fine with any approach, as the one I just outlined above as my current understanding makes perfect sense. I think the biggest question of all of these is what purpose the `normalizedForm` object is serving at this point. Do we need it?

Reply:
> So are you creating this object here in vets-website, making sure it renders appropriately, then, if so, ensuring that content-build is outputting things to match?

That was how I originally envisioned working on these components. (Renderer -> `content-build` -> CMS) As I've gotten through more of these, I've found it actually easier to do the reverse. (CMS -> `content-build` -> Renderer) So, what purpose do these static files still serve then?

The biggest benefit for me is that I don't have to deal with `content-build` while I'm building out the schema logic for each new pattern. I can (manually) switch the `fetchMethod` on `fetchFormConfig` and use the dummy files to make sure everything is rendering correctly locally. As you mentioned, `employmentQuestionnaire` is there to compare against the Employment Questionnaire design file. `normalizedForm` is there for me to test the pattern variations that `employmentQuestionnaire` does not cover. (What does Identification Information look like without Service Number, for example.) I'm also using them as fixtures within my test files. They probably need to be moved/removed at some point in the future, but for now, I still find them useful for quickly mocking what `content-build` would spit out.